### PR TITLE
Feature/44776 test support api

### PR DESCRIPTION
**Pull request by jon-shipley**
Initial test-support api for low environments.

### CODE REVIEW

File: tslib/src/functions/util-test-support-api/index.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+const httpTriggerFunc: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
+  if (!config.DevTestUtils.TestSupportApi) {
+    context.log('exiting as not enabled (default behaviour)')
+    return
+  }
+  // Respond in 230 seconds or the load balancer will time-out
+  // https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-http-webhook-trigger?tabs=javascript#limits
+  const start = performance.now()
+  const entity = context.bindingData.entity
```
Comment (GuyHarwood) [inline tslib/src/functions/util-test-support-api/index.ts:24]: maybe i've not had enough coffee yet, but how is this entity property populated?  the only binding we have is an incoming http request 🤔

**Review by GuyHarwood — APPROVED** [review]
LGTM

nice clean style on the api implementation 💯 

should be standard going forward

File: tslib/src/functions/util-test-support-api/index.ts (hunk 1 above)
Comment (jon-shipley) [reply tslib/src/functions/util-test-support-api/index.ts:24]: The `entity` bit is defined as an alpha in the `route` property of the function.json. 

`      "route": "test-support/{entity:alpha}"`

Comment (GuyHarwood) [reply tslib/src/functions/util-test-support-api/index.ts:24]: ah yes, overlooked the route stuff in the function definition. thanks!

**Pull request merged**